### PR TITLE
[bitnami/fluentd] Use different securityContext for aggregator and forwarder pods

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

**Description of the change**

This PR allows using different securityContexts for aggregator and forwarder pods. It also sets the **UID** and **fsGroup** for forwarder pods to **0** by default, so it can be configured to retrieve system-wide logs.

**Benefits**

Configuring Fluentd to retrieve system-wide logs.

**Possible drawbacks**

None

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/fluentd/values.yaml [hunk 1]
```diff
@@ -39,6 +39,15 @@ clusterDomain: cluster.local
...
   enabled: true
+
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 0
+    fsGroup: 0
+
```
Comment (alemorcuq) [inline bitnami/fluentd/values.yaml]: On the `README` you are saying that the default values are 1001. I assume the `README` is wrong.

File: bitnami/fluentd/values-production.yaml [hunk 2]
```diff
@@ -187,6 +196,14 @@ aggregator:
   ##
   replicaCount: 2
 
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 1001
+    fsGroup: 1001
```
Comment (sameersbn) [inline bitnami/fluentd/values-production.yaml]: in the `values.yaml` this is configured to have uid `0`, while in the `values-production,yaml` it's `1001`. Was it intended to be this way?

File: bitnami/fluentd/values-production.yaml [hunk 3]
```diff
@@ -39,6 +39,15 @@ clusterDomain: cluster.local
...
   enabled: true
+
+  ## K8s Security Context for forwarder pods
+  ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  ##
+  securityContext:
+    enabled: true
+    runAsUser: 1001
+    fsGroup: 1001
+
```
Comment (alemorcuq) [inline bitnami/fluentd/values-production.yaml]: This is different than in the `values.yaml` file, if that is intended please document the differences on the README.

File: bitnami/fluentd/values-production.yaml (hunk 2 above)
Comment (juan131) [reply bitnami/fluentd/values-production.yaml]: It was not intended. Both should be using `0`.

**Pull request merged**